inc/maze.h

```diff
@@ -21,29 +21,127 @@
 # define MAZE_E 4
 # define MAZE_W 8
 
+/**
+ * @brief Magic number from RAND_MAX
```

Comment:
```suggestion
 * @brief Maximum random value possible
```

This is not a magic number